### ISSUE TITLE
Maven build error

### ISSUE THREAD

**Issue by juliankeppel** [qubole/kinesis-sql#67]
I want to test your connector and first tried to build it with maven. JDK version is 13.0.2. I did exactly like you decribed in the README:
```
git clone...
git checkout 2.4.0
mvn clean install -DskipTests
```

But I get the following error (also on other branches like master branch):
```
[ERROR] Failed to execute goal org.apache.maven.plugins:maven-compiler-plugin:3.7.0:compile (default-compile) on project spark-sql-kinesis_2.11: Compilation failure: Compilation failure:
[ERROR] Source option 5 is no longer supported. Use 7 or later.
[ERROR] Target option 5 is no longer supported. Use 7 or later.
```

It seems that the maven-compiler-plugin has Java version 5 as default setting. But I wonder if I'm the only one with this issue...

If I set those versions manually in the `pom.xml` like this:
```
...
<configuration>
  <source>1.8</source>
  <target>1.8</target>
</configuration>
...
```

I get the following error: `java.lang.NoClassDefFoundError: javax/tools/ToolProvider`

Can anyone point me to the right direction what's wrong here? Thanks!

**Comment by juliankeppel** [comment]
I found a tip to add 
```
<configuration>
  <args>
    <arg>-nobootcp</arg>
   </args>
</configuration>
```
to scala-maven-plugin configuration and it works...

But I had to make two changes in the `pom.xml` to get it work. Maybe that's an issue though.

**Comment by itsvikramagr** [comment]
Thanks @juliankeppel for reporting and providing a workaround. I will look into it

**Comment by itsvikramagr** [comment]
@juliankeppel - I have fixed the issue with the fat jar. Checkout the latest master branch.

Can you please start a PR for the build error. We can discuss it there.

**Comment by juliankeppel** [comment]
> 
> 
> @juliankeppel - I have fixed the issue with the fat jar. Checkout the latest master branch.
> 
> Can you please start a PR for the build error. We can discuss it there.

Fat jar works, thank you!

I will start a PR for the Maven build problem.

**Comment by itsvikramagr** [comment]
resolved via #68
